google/gax/grpc.py

```diff
@@ -58,6 +58,10 @@
 """Maps strings used in client config to gRPC status codes."""
 
 
+NAME_STATUS_CODES = dict([(v, k) for (k, v) in STATUS_CODE_NAMES.items()])
+"""Inverse map for STATUS_CODE_NAMES"""
```

Comment:
Ah -- guessing you mean the former. `config.py` should have an alias, rather than recomputing the inverse. I'll fix it.